group_vars/docker-commons.yml.sample

```diff
@@ -18,3 +18,8 @@ dummy:
 #bootstrap_dirs_owner: "64045"
 #bootstrap_dirs_group: "64045"
 
+# The number of times to retry the docker image pull
```

Comment:
Remove this file, you edited `roles/ceph-defaults/defaults/main.yml` so you got `group_vars/rhcs.yml.sample` and `group_vars/all.yml.sample` generated. This is a leftover.